<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> QDialog {
                background-color: #f7f9f9;
}
            QLabel {
                font-family: 'Segoe UI', sans-serif;
                font-size: 15px;
                color: #000000;
                font-weight: bold;
            }
            QLineEdit, QComboBox {
                padding: 10px;
                border: 2px solid #03254c;
                border-radius: 8px;
                font-size: 15px;
                background-color: white;
                font-family: 'Segoe UI', sans-serif;
            }
            QLineEdit:focus, QComboBox:focus {
                border: 2px solid #03254c;
                outline: none;
            }
            QPushButton {
                background-color: #1167b1;
                color: white;
                padding: 10px 15px;
                border-radius: 8px;
                font-size: 14px;
                font-weight: bold;
                border: none;
                font-family: 'Segoe UI', sans-serif;
            }
            QPushButton:hover {
                background-color: #2a9df4;
            }
            QPushButton:pressed {
                background-color: #d0efff;
            }
            </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="login_gambar">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="username_input"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Passoword</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="password_input"/>
     </item>
     <item>
      <widget class="QLabel" name="error_label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btn_login">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
